### DISCUSSION TITLE
Welcome to StichHub Discussions!

### DISCUSSION THREAD

**Discussion by UBA-GCOEN** [UBA-GCOEN/StichHub#126] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Siddhant-Patil0203** [comment]
**Hello** Everyone, 👋
I am **Siddhant Patil** currently a Pre-final year **CSE Student** 🧑‍💻, and I am the **Project lead** ✨ of this project, 
It will be great to work with amazing developers 👩‍💻👨‍💻 all around the work and make **StichHub** a successful project together 🤝🌍

**Comment by LadeHarshal** [comment]
Hi everybody 👋
I am Harshal lade CSE student💻 at GCOEN and a part of this project StichHub.
I look forward to working with all you wonderful developers , let's learn and grow together.

**Comment by Naresh-chandanbatve** [comment]
Hello everyone 👋!
This is Naresh Chandanbatve pre-final year undergrad, aspiring DevOps Engineer and Open Source enthusiast. 

You are welcomed to the StichHub community feel free to reach me out or ping me for any help or suggestions.
